openhands/core/config/app_config.py

```diff
@@ -14,6 +15,21 @@
 from openhands.core.config.security_config import SecurityConfig
 
 
+def _get_or_create_jwt_secret() -> str:
+    config_dir = os.path.dirname(os.path.abspath(__file__))
+    secret_file = os.path.join(config_dir, '.jwt_secret')
```

Comment:
let's use the `file_store` for this actually--we're going to be adding that to the default `docker run` instructions as part of the config-saving and multi-conversation, so we might as well do it here.

In more detail:
* this should use the file store to look up and save the JWT file
* the Docker image should set env vars that point the file_store to `/.openhands` by default
* the README `docker run` should add `-v ~/.openhands:/.openhands` (and anywhere else we document `docker run`)

That'll make this work for everyone, and will set us up for all the other persistent state we're working on